aws/resource_aws_cloudtrail.go

```diff
@@ -300,6 +358,13 @@ func resourceAwsCloudTrailUpdate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	if d.HasChange("event_selector") {
```

Comment:
`d.HasChange` here will return true on resource creation (as `resourceAwsCloudTrailCreate` calls `resourceAwsCloudTrailUpdate`), so that means `PutEventSelectors` is called twice. You can use `!d.IsNewResource() && d.HasChange("event_selector")` to prevent that 👍